<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/eyeimage/logonomy-1687629985334.jpeg</normaloff>:/eyeimage/logonomy-1687629985334.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget:background-color(0,0,0,0);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-30</y>
     <width>531</width>
     <height>500</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>280</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,80);
border-top-left-radius: 50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Loginimage/pxfuel.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>240</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#b4d8ff;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LoginLogo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>69</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0,0,0,200);</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameinput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom: 2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordinput">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>215</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom: 2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">


QPushButton#LoginButton{
background-color: qlineargradient(spread:pad,x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11,131,120,219),stop:1 rgba(85,98,112,226));
color:rgba(255,255,255,210);
border-radius:5px;
}
QPushButton#LoginButton:hover{
background-color: qlineargradient(spread:pad,x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(150,123,111,219),stop:1 rgba(85,81,84,226));

}
QPushButton#LoginButton:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(150,123,111,255);

}</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUpButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#SignUpButton{
background-color: qlineargradient(spread:pad,x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11,131,120,219),stop:1 rgba(85,98,112,226));
color:rgba(255,255,255,210);
border-radius:5px;
}
QPushButton#SignUpButton:hover{
background-color: qlineargradient(spread:pad,x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(150,123,111,219),stop:1 rgba(85,81,84,226));

}
QPushButton#SignUpButton:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(150,123,111,255);

}
</string>
    </property>
    <property name="text">
     <string>SignUp</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorlable">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="errorlable_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Captcha">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad,x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11,131,120,219),stop:1 rgba(45,80,70,255));
color:rgba(255,255,255,210);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Recaptcha">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom: 2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>ReCaptcha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RECAPTCHAOK">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">


QPushButton#RECAPTCHAOK{
background-color: qlineargradient(spread:pad,x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11,131,120,219),stop:1 rgba(85,98,112,226));
color:rgba(255,255,255,210);
border-radius:5px;
}
QPushButton#RECAPTCHAOK:hover{
background-color: qlineargradient(spread:pad,x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(150,123,111,219),stop:1 rgba(85,81,84,226));

}
QPushButton#RECAPTCHAOK:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgba(150,123,111,255);

}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SuggestionList">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#b4d8ff;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Seepasswordbutton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/eyeimage/pngwing.com.png</normaloff>
      <normalon>:/eyeimage/pngwing.com.png</normalon>:/eyeimage/pngwing.com.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="usericon">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>155</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Loginimage/pngwin.com</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="passwordicon">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>218</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Loginimage/pngwing.com(1).png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="telegramlink">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue;
background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>click here</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>231</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="alreadySigned">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>You have already signedin</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_text_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
